qiskit_nature/mappers/second_quantization/qubit_mapper.py

```diff
@@ -35,40 +39,42 @@ def supports_particle_type(self, particle_type: ParticleOp) -> bool:
         Returns:
             A boolean indicating whether the queried particle-type is supported.
         """
-        return isinstance(particle_type, FermionicOp)
+        raise NotImplementedError()
 
+    @abstractmethod
     def map(self, second_q_op: ParticleOp) -> PauliSumOp:
-        """Maps a `SecondQuantizedOp` to a `PauliSumOp` using the Jordan-Wigner
-        fermion-to-qubit mapping.
+        """Maps a `ParticleOp` to a `PauliSumOp`.
 
         Args:
-            second_q_op: the `SecondQuantizedOp` to be mapped.
+            second_q_op: the `ParticleOp` to be mapped.
+
         Returns:
             The `PauliSumOp` corresponding to the problem-Hamiltonian in the qubit space.
+        """
+        raise NotImplementedError()
+
+    @staticmethod
+    def mode_based_mapping(second_q_op: ParticleOp,
```

Comment:
I am okay with leaving things as they are here now. But I would like to address my point about a "label-translation"-based implementation of these mappers. I agree, that this is not a high priority now, but when we will refactor BKSF, we might as well refactor these ones too, because it will likely help with the implementation of the "more difficult" BKSF mapper, if we first did these simple ones. No need to do it now, I just wanted to put this out here.

Reply:
Sure, the current set (what has been done) is based on sites. The BKSF is based on interactions between sites. I do not have enough understanding to know how much the former would inform the latter. As you have seen in the current codebase they are two very different things even though they are based on the same integrals.